api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -207,6 +207,70 @@ message RedisProxy {
     repeated string commands = 4;
   }
 
+  // FeatureConfig defines the configuration of all extra features.
+  message FeatureConfig {
+    // HotKey defines the configuration of hotkey feature.
```

Comment:
Can you describe more what this feature does? It's not clear from the docs what this means.